.circleci/config.yml

```diff
@@ -23,14 +23,39 @@ jobs:
           rustup target add wasm32-wasi --toolchain nightly-2021-04-24-x86_64-unknown-linux-gnu
           cd ./service
           ./build.sh
+      - persist_to_workspace:
```

Comment:
you don't need this. Just concatenate Build and Rust tests. Copying workspace takes ~30 seconds, too long: doesn't worth it.